docs/supervisor.md

```diff
@@ -0,0 +1,63 @@
+# Velero Plugin for vSphere in vSphere with Kubernetes Supervisor Cluster
+
+## Table of Contents
+1. [Compatibility](#compatibility)
+2. [Prerequisites](#prerequisites)
+3. [Install](#install)
+4. [Uninstall](#uninstall)
+5. [Backup](#backup)
+6. [Restore](#restore)
+
+## Compatibility
+
+| vSphere Version |                           vSphere CSI Version                          | Kubernetes Version | Velero Version | Velero Plugin for vSphere Version |
+|:---------------:|:----------------------------------------------------------------------:|:------------------:|:--------------:|:---------------------------------:|
+|  vSphere 7.0 U1c/P02 + ESXi 7.0 U1c/P02 | Bundled with vSphere |     Bundled with vSphere    |     v1.5.1     |         v1.1.0 and higher         |
```

Comment:
v1.5.1 -> v1.5.1 and higher

Reply:
Bundled with vSphere  -> Bundled with vSphere  (v1.16-v1.19)

Reply:
> v1.5.1 -> v1.5.1 and higher

Leave it as it is based on offline discussion.